migration-engine/connectors/sql-migration-connector/src/flavour/mysql.rs

```diff
@@ -46,6 +46,11 @@ impl MysqlFlavour {
 
 #[async_trait::async_trait]
 impl SqlFlavour for MysqlFlavour {
+    async fn acquire_lock(&self, connection: &Connection) -> ConnectorResult<()> {
+        // https://dev.mysql.com/doc/refman/8.0/en/locking-functions.html
+        Ok(connection.raw_cmd("SELECT GET_LOCK('prisma_migrate', 60)").await?)
```

Comment:
The magic number here looks like a timeout of 60 secs. This is inconsistent with the duration of the `Postgres` timeout (10 secs). Is there a reason why these values differ?

I'm not familiar with the abstractions; but I think copying the postgres approach here is probably right, if not configuring it from a central place:

```rust
// either in src/flavour/mysql.rs, or some /src/shared.rs file
const ADVISORY_LOCK_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(10);
```

Reply:
This is indeed an oversight — I don't know what a good default there is, so we'll rely on user feedback I'll align it to 10 seconds for now.